modinfo.lua

```diff
@@ -8,8 +8,8 @@ return {
   onlyLocal = true,
   depend = {
       -- For developing base chobby, switch out dependency
-      'rapid://chobby:test', --this uses rapid pinned chobby
-      --'Chobby $VERSION', -- this specifies chobby.sdd working path
+      -- 'rapid://chobby:test', --this uses rapid pinned chobby
+      'Chobby $VERSION', -- this specifies chobby.sdd working path
```

Comment:
@Beherith
Will doing this mess things up for players when they update?

Reply:
Yup, please undo

Reply:
Yep, im sure this is an accidental part of the commit.

Reply:
Yes, sorry about that ...

Reply:
Fixed